src/sass/theme/_understrap.scss

```diff
@@ -58,10 +58,78 @@ body {
 	display: block;
 }
 
-.widget_categories,
 .widget_archive {
+
 	select {
-		@extend .form-control;
+		display: block;
+		width: 100%;
+
+		@if variable-exists("bootstrap4") { // replicates relevant parts of .form-control
```

Comment:
Which docs are you referring to? Why is `.form-control` wrong on a `<select>` for Bootstrap 4?

Reply:
I guess I was thinking `.custom-select` is the preferred select element class for BS4, but maybe `.form-control` is actually fine since that's what we've been using. Custom select just makes the `<select>` element more consistent across different browsers.

https://getbootstrap.com/docs/4.6/components/forms/#form-controls
https://getbootstrap.com/docs/4.6/components/forms/#select-menu

Reply:
I personally use form-control in order to skip the custom form styles at all. I think Understrap uses both `.form-control` and `.custom-select`.